recomendator/recomendations/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
+# Register your models here.
```

Comment:
Is this necessary?

Reply:
I'll add a model to the service as 'Recomendations' to persist the reomendations made, the admin could be used to check and modifiy them, but I don't have huge pros with this.
At the end is the auto generated code by django